circle.yml

```diff
@@ -36,14 +39,22 @@ test:
         parallel: true
     - cd $SRCDIR; make testrunner:
         parallel: true
-    - cd $SRCDIR/test; ./gce.sh setup && eval $(./gce.sh hosts) && ./setup.sh:
-        parallel: true
-    - cd $SRCDIR/test; eval $(./gce.sh hosts); export COVERAGE=true; ./run_all.sh:
-        parallel: true
+    #- cd $SRCDIR/test; ./gce.sh setup && eval $(./gce.sh hosts) && ./setup.sh:
+    #    parallel: true
+    #- cd $SRCDIR/test; eval $(./gce.sh hosts); export COVERAGE=true; ./run_all.sh:
+    #    parallel: true
+    #    timeout: 300
+    - cd $SRCDIR/test; ./aws.sh setup && eval $(./aws.sh hosts) && ./setup.sh:
+        parallel: false
```

Comment:
Why `parallel false` ?  This means we only run smoke-tests on one side of the build, which slows down the overall completion.